.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedParser.cs

```diff
@@ -75,6 +63,19 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refTime)
                 er.Text = er.Text.Substring(sinceMatch.Length);
                 modStr = sinceMatch.Value;
             }
+            else if (er.Type.Equals(Constants.SYS_DATETIME_DATEPERIOD) && Config.YearRegex.Match(er.Text).Success)

```

Comment:
Why this order change? Please add a comment in the code describing the flow and need; in case this is strictly necessary. To avoid others breaking it in the future by mistake.